towncrier/newsfragments/1194.misc

```diff
@@ -0,0 +1 @@
+Switch to bcrypt_sha256, remove PASSWORD_SCHEME and replace it with CREDENTIAL_ROUNDS
```

Comment:
```suggestion
Switch to bcrypt_sha256, replace PASSWORD_SCHEME with CREDENTIAL_ROUNDS and update existing hashes on first login
```

This PR fixes multiple issues, so it's a little bit difficult to express it properly in one newsfragment and I think it sometimes makes sense to also link to other issues like you did. But imo at least one of them should link to this pr, so users can easily find the actual change.

I'd suggest to link this line to #1753, because in case someone has problems with changes introduced by this PR after upgrading they'll probably suspect this change to be the reason for it after grepping for `password` or `login`.

Reply:
That's 537bb3c